pyspectral/tests/test_rayleigh.py

```diff
@@ -201,6 +203,18 @@ def test_get_reflectance(self, get_effective_wvl,
             sun_zenith, sat_zenith, azidiff, 'M2', blueband)
         self.assertTrue(np.allclose(retv, TEST_RAYLEIGH_RESULT2))
 
+        # Test the Rayleigh reduction code
+        sun_zenith = np.array([70., 65., 60.])
+        sat_zenith = np.array([65., 49., 26.])
+        azidiff = np.array([30., 140., 130.])
+        blueband = np.array([14., 12., 8.])
+        retv = self.viirs_rayleigh.get_reflectance(
+            sun_zenith, sat_zenith, azidiff, 'M2', blueband, reduce_hizen=60, reduce_strength=0.5)
+        self.assertTrue(np.allclose(retv, TEST_RAYLEIGH_RESULT_R1))
+        retv = self.viirs_rayleigh.get_reflectance(
+            sun_zenith, sat_zenith, azidiff, 'M2', blueband, reduce_hizen=60, reduce_strength=1.5)
+        self.assertTrue(np.allclose(retv, TEST_RAYLEIGH_RESULT_R2))
+
```

Comment:
Could you put the reduction part in a separate test function? The test functions are already too long. Preferably a test-function should test only one thing.